Fulfillment/Operation/01_ManageDispatchNotes.md

```diff
@@ -0,0 +1,140 @@
+[!!User Interface Dispatch notes](../UserInterface/01a_List.md)
+[!!User Interface Logging](../UserInterface/02a_Connections.md)
+[!!User Interface Packaging types](../UserInterface/03b_PackagingTypes.md)
+[!!Check a shipment](./02_CheckShipment.md)
+[!!Manage the connections](../Integration/01_ManageConnections.md)
+
+
+
+# Manage the dispatch notes
+
+Dispatch notes are shipment requests generated by the system when a delivery note has been issued in the *Order management* module. The dispatch note is than sent to the order fulfiller, who will in turn arrange the shipment.
+
+The communication between the *Actindo Core1 Platform* and the fulfiller's internal system takes place via a connection driver. The *Actindo Core1 Platform* sends the fulfiller the information they need based on the information provided in the delivery note, which has been previously mapped via the ETL function in the *DataHub* module. The fulfiller, on the other hand, sends back a confirmation message to update the shipment status accordingly during all stages of the fulfillment process.   
+
+The *Fulfillment* module works together with the *Workflows* module. Once the *Fulfillment* module has been configured, a workflow must be created for the corresponding business process, which runs automatically in the *Workflows* module. The *Fulfillment* module is therefore designed to monitor the fulfillment process but not to steer it. For this purpose, the *Workflows* module should be used.
+
+[comment]: <> (Link auf Operation/ManageWorkflows? Evtl. extra Info in Integration?)
```

Comment:
Meinetwegen ein related link auf Workflows; was meinst du mit extra info in Integration?